ui/admin/tests/acceptance/credential-store/credentials/selectors.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const FIELD_TYPE_SSH = '[value=ssh_private_key]';
+export const FIELD_TYPE_USERNAME_PASSWORD = '[value=username_password]';
+export const FIELD_TYPE_JSON = '[value=json]';
+export const EDITOR = '[data-test-code-editor-field-editor]';
```

Comment:
Open to change the name of this selector 😉 . Initially I though on calling it EDITOR_CONTAINER but beign a domain specific selector file end up with this.

Reply:
I'm ok with this selector name!